tests/test_split.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from typing import Any, Mapping
+
+from snuba import state
+from snuba.datasets.factory import get_dataset
+from snuba.query.query import Query
+from snuba.request import Request
+from snuba.request.request_settings import RequestSettings
+from snuba.split import split_query
+
+
+def setup_function(function):
+    state.set_config('use_split', 1)
+
+
+test_data_no_split = [
+    "events",
+    "transactions",
+    "groups",
+]
+
+
+@pytest.mark.parametrize("dataset", test_data_no_split)
+def test_no_split(dataset: str):
+    query = Query({
+        "selected_columns": ["event_id"],
+        "conditions": [""],
+        "orderby": "event_id",
+        "sample": 10,
+        "limit": 100,
+        "offset": 50,
+    })
+
+    @split_query
+    def do_query(dataset, request: Request, timer):
```

Comment:
Just curious: is there a reason for putting types on some of the params and not all?

Reply:
fixed.